<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyRadioButton</class>
 <widget class="QWidget" name="MyRadioButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radioOption01">
      <property name="text">
       <string>Choose 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioOption02">
      <property name="text">
       <string>Choose 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioOption03">
      <property name="text">
       <string>Choose 3</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
